### ISSUE TITLE
Tooltip DAP Violation (AVT 1)

### ISSUE THREAD

**Issue by johnharan** [IBM/carbon-components-angular#262]
The 'aria-describedby' attribute which is binded to the host element of the TooltipDirective (selector 'ibmTooltip') references a DOM element which only exists on hover of the host element.
So when running a DAP scan, the following DAP violation shows if the host element is not being hovered:

> The id identifier specified for WAI-ARIA property identifier on element identifier is not valid. See rule help for remediation guidance. 

>The id tooltip-1 specified for WAI-ARIA property aria-describedby on element span is not valid. See rule help for remediation guidance.

https://aat.w3ibm.mybluemix.net/token/8e33e901-068f-4fe0-941a-7a5aa0af44ca/6f511ffe-45fc-48e3-ac7f-1d0d8a6975c7/latest///doc/w3/help/en-US/idhi_accessibility_check_g1077.html

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> What browser are you working in?

chrome Version 70.0.3538.77

> What version of the Carbon Design System are you using?

"carbon-components": "9.48.0",
"carbon-components-angular": "2.0.6",

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

Screenshot
![example](https://user-images.githubusercontent.com/8806230/48350915-810b3d80-e680-11e8-9188-e0560ac9c3e4.png)
